### DISCUSSION TITLE
How to make Motrix or other app that has a system tray always running in the background?

### DISCUSSION THREAD

**Discussion by sauntor** [elementary/triage#273] [Q&A]
As the title says, some apps need to stay in the background to provide it's functionality,  i.e., Motrix need to be running to take over  the file downloading of chrome. The app may be installed via `apt` or `flatpak`. So, how can we make it possiable?

**Comment by sauntor** [comment]
On other systems, which have a system tray,  if we close the main window of a app, it can stay in the system tray, but on `Elementary OS`, the app will exit entirely! That's the actually problem I want to resolve!

**Comment by lenemter** [comment]
You can install system tray for elementary OS, the most recent one is https://github.com/MvBonin/wingpanel-community-indicators

**Reply by sauntor** [reply]
`System Tray` may be necessary, as mentioned in https://github.com/elementary/triage/issues/334

**Comment by secondlnl** [comment]
elementary OS intentionally doesn't support system trays [see this wingpanel issues#96 comment by Cassidy](https://github.com/elementary/wingpanel/issues/96#issuecomment-401407354). 
For a application to run in the background it is my understanding that the application needs to support a [freedesktop portal for background activity](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.Background).  (Maybe needs something more but I am not an developer)
You could try to reach the developer for said application.
